src/vs/workbench/contrib/positronConsole/browser/positronConsoleActions.ts

```diff
@@ -427,7 +430,7 @@ export function registerPositronConsoleActions() {
 
 			// Ask the Positron console service to execute the code. Do not focus the console as
 			// this will rip focus away from the editor.
-			if (!await positronConsoleService.executeCode(languageId, code, false, allowIncomplete)) {
+			if (!await positronConsoleService.executeCode(languageId, code, false, allowIncomplete, opts.runtimeCodeExecutionMode)) {
```

Comment:
While I was making the change to the `positronConsoleService` to respect the code execution mode I saw this action was using the service and added support for it. Looking back at this change, I don't think we want to allow the user to specify what mode the code is executed in. This should probably always run in interactive mode.

Thoughts?

Reply:
Alrighty, I will leave this as is then! Thanks for chiming in :)